wordpress-mobile/WordPress-iOS
# Removes legacy editor

This PR removes the legacy editor from the app. It also removes the reference to the WPWordPressEditor pod that is no longer in use.

To test:
 - Start the app
 - Check that for editing/updating post or pages the new Aztec editor is being used.
